allure-commandline/src/main/java/ru/yandex/qatools/allure/logging/ANSIConsoleAppender.java

```diff
@@ -1,78 +0,0 @@
-package ru.yandex.qatools.allure.logging;
-
-import org.apache.log4j.ConsoleAppender;
-import org.apache.log4j.Level;
-import org.apache.log4j.Priority;
-import org.apache.log4j.spi.LoggingEvent;
-
-/**
- * @author Artem Eroshenko <eroshenkoam@yandex-team.ru>
- */
-public class ANSIConsoleAppender extends ConsoleAppender {
```

Comment:
BTW I can add the `ANSIConsoleAppender` back, but disable it by default. So you can switch it on if you need